lib/build.rs

```diff
@@ -40,6 +40,11 @@ fn generate_module_files(proto_files: Vec<FileDescriptorProto>) {
         }
     }
 
+    // Sort the modules by name to avoid non-deterministic builds
+    modules.sort_by_key(|it| it.3.to_owned());
```

Comment:
Already done in https://github.com/VirusTotal/yara-x/commit/4a9e0cc19a14c40e0a73e0f6f04559c172cd57e1

Reply:
removed in [cca5adf](https://github.com/VirusTotal/yara-x/pull/183/commits/cca5adfaeb3671482a47f5b0a1a8e3f7d2a70fe8)